### ISSUE TITLE
Contexts unit tests failing in Travis w/ OFI master

### ISSUE THREAD

**Issue by davidozog** [Sandia-OpenSHMEM/SOS#523]
With SOS 1.4.0-dev and the libfabric master branch, the "many-ctx" and "web" unit tests fail with the following stack trace (and at least 2 processes):
```
#0  0x00007ffff72551ec in sock_pe_progress_tx_entry (pe=0x7ffff7faf010, tx_ctx=0x625c90, pe_entry=0x7ffff7fce520) at ../prov/sockets/src/sock_progress.c:1926
#1  0x00007ffff7257115 in sock_pe_progress_tx_ctx (pe=0x7ffff7faf010, tx_ctx=0x625c90) at ../prov/sockets/src/sock_progress.c:2472
#2  0x00007ffff7257b1c in sock_pe_progress_thread (data=0x7ffff7faf010) at ../prov/sockets/src/sock_progress.c:2648
#3  0x00007ffff6ddf6ba in start_thread (arg=0x7ffff6bd3700) at pthread_create.c:333
#4  0x00007ffff75af3dd in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:109
```
Note that this does *not* occur on libfabric ~v1.5.1 and earlier, and it has so far only manifested in Linux with the sockets provider.  I think the problem arises from somewhere around [this commit in libfabric](https://github.com/ofiwg/libfabric/commit/74571f240a6b989ac71808ab148fdae362d297f8).

**Comment by davidozog** [comment]
libfabric issue [#3557](https://github.com/ofiwg/libfabric/issues/3557)

**Comment by davidozog** [comment]
Appears to be fixed in libfabric PR [#3584](https://github.com/ofiwg/libfabric/pull/3584)

**Comment by jdinan** [comment]
Hooray!  Let's leave this ticket open as a reminder to move Travis back to libfabric master once the above PR is merged.
